manifests/g/GrafanaLabs/GrafanaEnterprise/8.0.2/GrafanaLabs.GrafanaEnterprise.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate
+
+PackageIdentifier: GrafanaLabs.GrafanaEnterprise
+PackageVersion: 8.0.2.0
+PackageLocale: en-US
+Publisher: Grafana Labs
+PackageName: Grafana Enterprise

```

```suggestion
PackageName: GrafanaEnterprise
```
To match whatever is shown in Control Panel.